While compiling with g++ 8.2.0 std::ignore cannot be found

Using g++ 8.2.0 to compile Cajita leads to an error, when std::ignore cannot be
resolved in src/Cajita_UniformDimPartitioner.cpp:

```
/home/halver/install/Cajita/src/Cajita_UniformDimPartitioner.cpp:21:10: error: ‘ignore’ is not a member of ‘std’
     std::ignore = global_cells_per_dim;
          ^~~~~~
```

--- a/src/Cajita_UniformDimPartitioner.cpp
+++ b/src/Cajita_UniformDimPartitioner.cpp
@@ -16,10 +16,8 @@ namespace Cajita
 //---------------------------------------------------------------------------//
 std::vector<int> UniformDimPartitioner::ranksPerDimension(
     MPI_Comm comm,
-    const std::vector<int>& global_cells_per_dim ) const
+    const std::vector<int>& ) const
 {
-    std::ignore = global_cells_per_dim;
-
     int comm_size;
     MPI_Comm_size( comm, &comm_size );
 

